sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/tests/TestListener.cs

```diff
@@ -78,12 +77,52 @@ public async Task TestSubscribe(string evt, string functionName)
             Assert.AreEqual("validation-code", JObject.Parse(content)["validationResponse"].ToString());
         }
 
+        [Test]
```

Comment:
If I was mistaken about being mutually exclusive, should we consider a test case where both are set?

Reply:
Nope, you were correct that they are mutually exclusive. The runtime would throw if multiple trigger attributes are present.